modules/gdscript/gdscript_analyzer.h

```diff
@@ -135,6 +135,7 @@ class GDScriptAnalyzer {
 	Ref<GDScriptParserRef> get_parser_for(const String &p_path);
 	void reduce_identifier_from_base_set_class(GDScriptParser::IdentifierNode *p_identifier, GDScriptParser::DataType p_identifier_datatype);
 #ifdef DEBUG_ENABLED
+	void enforce_static_typing(GDScriptParser::Node *p_node, String p_identifier);
```

Comment:
```suggestion
	void enforce_static_typing(GDScriptParser::Node *p_node, const String &p_identifier);
```
Prefer to pass by reference here, cheaper